test/actor_test.py

```diff
@@ -2380,3 +2311,259 @@ def inc(self, duration):
     for _, result_id_list in result_ids.items():
         results = list(range(1, len(result_id_list) + 1))
         assert ray.get(result_id_list) == results
+
+
+def test_checkpointing(ray_start_regular, ray_checkpointable_actor_cls):
+    """Test actor checkpointing and restoring from a checkpoint."""
+
+    def kill_actor(actor):
```

Comment:
Maybe move this outside the function? The same code is repeated in a few other places.

Reply:
Do you mean the similar code in the `test_actor_reconstruction` test? that one is a bit different. In that function, we need to get the pid before calling other methods. So I cannot re-use this function for that test.

Reply:
It looks to me that `kill_actor` in `test_remote_checkpointing`, `test_checkpointing_save_exception`, and `test_checkpointing_load_exception` are all the same.

Reply:
ah, I didn't notice Stephanie added those tests. Thanks.